### ISSUE TITLE
Chart template for Module.spec.ambassador_id is not an array

### ISSUE THREAD

**Issue by crdueck** [emissary-ingress/emissary#4733]
**Describe the bug**
The `Module.spec.ambassador_id` field is documented that it must be an array: https://www.getambassador.io/docs/edge-stack/latest/topics/running/ambassador#ambassador-id

But the template in the chart is a quoted string: https://github.com/emissary-ingress/emissary/blob/master/charts/emissary-ingress/templates/module.yaml#L15

So it's not possible to render an array, either with `[`-paren or yaml array syntax.

**To Reproduce**
1. Download `emissary-ingress` helm chart: `helm pull datawire/emissary-ingress`
2. Template chart with `AMBASSADOR_ID` env var set: `helm template . --set "env.AMBASSADOR_ID=custom-id"`
4. Rendered output with unrelated metadata labels removed:
```yaml
# Source: emissary-ingress/templates/module.yaml
apiVersion: getambassador.io/v2
kind: Module
metadata:
  name: ambassador
  namespace: default
spec:
  ambassador_id: "custom-id"
  config:
    diagnostics:
      allow_non_local: true
      enabled: false
```

**Expected behavior**
The `Module.spec.ambassador_id` field should be an array as documented. So I expected
```yaml
spec:
  ambassador_id: [ "custom-id" ]
```
or 
```yaml
spec:
  ambassador_id: 
    - "custom-id"
```

**Versions (please complete the following information):**
 - Emissary-ingress chart version v8.3.0

**Comment by akundrock** [comment]
I have this issue too but decided to use 
`ambassador_id: 
    {{ toYaml .Values.env.AMBASSADOR_ID }}`

because the quote in the current code was still causing issues for me.
Hope to see this PR merged!

**Comment by cindymullins-dw** [comment]
Fixed in v8.5.1.
